src/test/java/com/google/sps/integration/ElectionInfoIT.java

```diff
@@ -0,0 +1,33 @@
+package com.google.sps.integration;
+
+import org.junit.*;
```

Comment:
Don't use * in import per google style guide. 

The reason why is that it's harder to refactor, less efficient, and less convenient for readers (who often want to see where something is imported from).

Reply:
This was done by the auto-compiler, fixing now.